### DISCUSSION TITLE
How do I stop NeoTree from closing after opening file on startup?

### DISCUSSION THREAD

**Discussion by Gauserr** [nvim-neo-tree/neo-tree.nvim#930] [Q&A]
Hi there, 

I have neotree set to 'open_current' so that when i do 'nvim .' it opens NeoTree rather than Netrw. Love this feature. However, when i then choose a file to open, it opens the file without keeping neotree open. 

is there a way that i can open a file and keep the tree open?

thanks a ton in advance

**Comment by pysan3** [comment]
Does this solution work for you as well?

https://github.com/nvim-neo-tree/neo-tree.nvim/discussions/919
